spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/SarifBugReporter.java

```diff
@@ -59,27 +74,49 @@ private void processInvocations(JSONWriter jsonWriter, @NonNull Map<URI, String>
         Invocation invocation = new Invocation(exitCode,
                 getSignalName(exitCode), exitCode == 0,
                 execNotifications, configNotifications);
-        jsonWriter.key("invocations").array().value(invocation.toJSONObject());
-        jsonWriter.endArray();
+        try {
+            Gson gson = new Gson();
+            jsonWriter.name("invocations").beginArray();
+            gson.toJson(invocation.toJsonObject(), jsonWriter);
+            jsonWriter.endArray();
+        } catch (IOException e) {
+            throw e;
+        }
```

Comment:
This try-block seems needless.
```suggestion
        Gson gson = new Gson();
        jsonWriter.name("invocations").beginArray();
        gson.toJson(invocation.toJsonObject(), jsonWriter);
        jsonWriter.endArray();
```